docs/pages/careers/staff-engineer-pigment-css.md

```diff
@@ -0,0 +1,130 @@
+We're well-positioned to make this happen, so we're looking for a talented engineer to join the team and help us get further beyond!
+
+### Why this is interesting
+
+The core components that will depend on this library are widely used, you will receive a lot of feedback on your work.
+The community has high expectations, they will push you to do better, every day.
+
+Our products empower React developers to build awesome applications faster – we see millions of developers on MUI's docs every year, one million a month.
+
+## The role
+
+### Key responsibilities
+
+- **Build**. You will provide solutions for integrating the library with various bundlers and frameworks. You will build new features and improve the DX of the library.
+- **Nurture community contributions**. You will provide guidance and direction to unlock the contributions of the community. Your time will often be way better spent doing this than fixing the problems yourself.
+- **Shape the product**. You will be laser-focused on the end goal. It's not about solving technical challenges but about the problems solved for the users.
+- **Keep technical debt in check**. You will make sure we can keep shipping features at a reasonable pace, align the team on "one way" of doing things and make sure engineers follow the conventions.
+- **Be a solver**. You will dig deep into arbitrarily complex problems and find an appropriate path forward. Some focus on a given area for long periods, others bounce from hotspot to hotspot as guided by organizational leadership.
+
+### The best parts of this job
+
+- **You'll be at the cutting edge of application development** — working on one of the fastest-growing CSS-in-JS libraries on the market.
+- **You'll be part of an active, open, friendly community** of developers that are excited about building awesome applications.
+- **Your role will be key to making MUI the go-to UI solution** for building applications, websites, and design systems that does not have to trade performance for a great developer experience.
+
+### The worst parts of this job
+
+- **Shifting context.**
+  You will necessarily have to shift context and dive into a different feature before the current one is done.
+  It may even be in an area of the codebase you're unfamiliar with or don't have a ton of understanding about.
+  It's fun, rewarding work, but it can be very challenging.
+- **We move quickly but don't sacrifice quality**.
+  We ship early, often, and quickly. You may not be initially comfortable with the cadence with which we ship high-quality features and improvements to end-users. By doing so, we sacrifice on solving each problem 100% in exchange for fast feedback. Solving 50-70% of the issue with quality should be enough for any given iteration. Our users quickly tell us when we haven't pushed a solution far enough.
+- **MUI has a large codebase**. You may bang your head against the wall at times, and then write tests to assist future you 😌.
```

Comment:
```suggestion
- **MUI has large codebases**. You may bang your head against the wall at times and then write tests to assist future you.
```

"MUI has a large codebase" in singular makes me read this as "Material UI has a large codebase". I think plural here resonates more with the idea of multiple products.